### PR TITLE
Gudemchuk Daria

### PR DESCRIPTION
**Pull request by darinagdm**

### CODE REVIEW

**Review by Dmitriy-NK — CHANGES_REQUESTED** [review]
1. График 4 (в большом наборе графиков) построен с нормированием к полному числу пассажиров. Предполагается сначала разбиение на подмножества а потом для каждого подмножества построение графика. у seaborn.kdeplot для этого используется аргумент "common_norm". 
2. В разделе "graph down by gender" отсутствует график пропорционального распределения "# Next adjust graph to display the proportions of survival by gender"  [пример](https://github.com/srk-practicum/2022-knu-yolo/issues/35)
3. И просьба добавить несколько предложений по своей оценке после "Add your summary about recieved data for raw value counts and proportionally" в том же разделе
4. К следующему разделу (Pclass) нет деления на выживших и не выживших, и добавить соответствующую оценку в вывод. [из мелкого предпочтительней использовать != 3 а не <=2 или <3]
5. Во всех confusion matrix метки(подписи heatmap) не соответствуют значениям, по умолчанию sklearn.metrics.confusion_matrix считает метку 0 - положительной, так собственный порядок можно задать в аргументе labels, [detailes](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)

**Review by Dmitriy-NK — CHANGES_REQUESTED** [review]
Замечания к части по анализу данных [здесь](https://github.com/srk-practicum/2022-knu-yolo/pull/17). Тренировка и оценка моделей [здесь](https://github.com/srk-practicum/2022-knu-yolo/pull/31)

**Review by Dmitriy-NK — COMMENTED** [review]
3-й график "Class Distribution" в наборе графиков построен с ошибкой

**Review by Dmitriy-NK — APPROVED** [review]
Approved.
